src/client/eppo-client-with-bandits.spec.ts

```diff
@@ -89,10 +92,22 @@ describe('EppoClient Bandits E2E test', () => {
             };
           });
 
-          const subjectAttributes = {
-            ...subject.subjectAttributes.numeric_attributes,
-            ...subject.subjectAttributes.categorical_attributes,
-          };
+          // TODO: handle already-bucketed attributes
```

Comment:
Future PR will allow folks to explicitly specify if attributes are numeric or categorical